### DISCUSSION TITLE
help on "cv2" module

### DISCUSSION THREAD

**Discussion by GiorgioRafael** [hacksider/Deep-Live-Cam#190] [Q&A]
did everything following the instructions and still cant run

![imagem_2024-08-11_163927613](https://github.com/user-attachments/assets/9307f103-2a7e-4c02-bdae-cd1cca48718b)

**Comment by GylanSalih** [comment]
Hey i hope your doing well :)
here https://github.com/hacksider/Deep-Live-Cam/discussions/110
a user posted a Install Video https://www.youtube.com/watch?v=MICScLVM1x8
Watch the video maybe you forget a step 
hope i can help you

**Reply by GiorgioRafael** [reply]
did step by step and got the same error as the following image
![image](https://github.com/user-attachments/assets/278529c1-dc53-4df3-b7e2-358e9843fc40)

**Reply by PolymicroDev** [reply]
Same here

**Comment by vlednerk** [comment]
> did everything following the instructions and still cant run
> 
> ![imagem_2024-08-11_163927613](https://private-user-images.githubusercontent.com/131801417/356901891-9307f103-2a7e-4c02-bdae-cd1cca48718b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1NzM1NjUsIm5iZiI6MTcyMzU3MzI2NSwicGF0aCI6Ii8xMzE4MDE0MTcvMzU2OTAxODkxLTkzMDdmMTAzLTJhN2UtNGMwMi1iZGFlLWNkMWNjYTQ4NzE4Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxM1QxODIxMDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wM2NmNGQxNDMyYTc0ZTVjOGY5MjY5NWJmOWNlMTlhNzBlZmMxZWZjZDQ1NGMwMjlkZWMzZTgxMTE0NjdkYjlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.aU3o1efC9uoWw79ySmdbh5RTCT322cVBItDFYybpCF4)

pip install opencv-python==4.8.1.78
